patch/0001-armhf-mandatory-configs-for-boot.patch

```diff
@@ -0,0 +1,201 @@
+From a961464da6c88aee4d94af4038c748a1e8a1f412 Mon Sep 17 00:00:00 2001
+From: Antony Rheneus <arheneus@marvell.com>
+Date: Thu, 17 Oct 2019 18:57:48 +0530
+Subject: [PATCH] armhf mandatory configs for boot
+
+---
+ debian/build/build_armhf_none_armmp/.config | 53 ++++++++++++++++-------------
+ 1 file changed, 30 insertions(+), 23 deletions(-)
+
+diff --git a/debian/build/build_armhf_none_armmp/.config b/debian/build/build_armhf_none_armmp/.config
+index e997934..69288eb 100644
+--- a/debian/build/build_armhf_none_armmp/.config
++++ b/debian/build/build_armhf_none_armmp/.config
+@@ -130,14 +130,14 @@ CONFIG_CGROUPS=y
+ CONFIG_PAGE_COUNTER=y
+ CONFIG_MEMCG=y
+ CONFIG_MEMCG_SWAP=y
+-# CONFIG_MEMCG_SWAP_ENABLED is not set
++CONFIG_MEMCG_SWAP_ENABLED=y
+ CONFIG_BLK_CGROUP=y
+ # CONFIG_DEBUG_BLK_CGROUP is not set
+ CONFIG_CGROUP_WRITEBACK=y
+ CONFIG_CGROUP_SCHED=y
+ CONFIG_FAIR_GROUP_SCHED=y
+ CONFIG_CFS_BANDWIDTH=y
+-# CONFIG_RT_GROUP_SCHED is not set
++CONFIG_RT_GROUP_SCHED=y
+ CONFIG_CGROUP_PIDS=y
+ CONFIG_CGROUP_FREEZER=y
+ CONFIG_CPUSETS=y
+@@ -207,7 +207,7 @@ CONFIG_PERF_USE_VMALLOC=y
+ CONFIG_PERF_EVENTS=y
+ # CONFIG_DEBUG_PERF_USE_VMALLOC is not set
+ CONFIG_VM_EVENT_COUNTERS=y
+-# CONFIG_COMPAT_BRK is not set
++CONFIG_COMPAT_BRK=y
+ CONFIG_SLAB=y
+ # CONFIG_SLUB is not set
+ # CONFIG_SLOB is not set
+@@ -1736,7 +1736,7 @@ CONFIG_TEGRA_AHB=y
+ #
+ # CONFIG_UEVENT_HELPER is not set
+ CONFIG_DEVTMPFS=y
+-# CONFIG_DEVTMPFS_MOUNT is not set
++CONFIG_DEVTMPFS_MOUNT=y
```

Comment:
>CONFIG_DEVTMPFS_MOUNT [](start = 1, length = 21)

why you need it?

Reply:
I will check this and update if not needed.

Reply:
Not needed, removed the change